domenetjenester/tilgangskontroll/src/main/java/no/nav/ung/sak/tilgangskontroll/integrasjon/pdl/dto/PipPerson.java

```diff
@@ -1,21 +1,7 @@
 package no.nav.ung.sak.tilgangskontroll.integrasjon.pdl.dto;
 
-import com.fasterxml.jackson.annotation.JsonAutoDetect;
-import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-@JsonFormat(shape = JsonFormat.Shape.OBJECT)
-@JsonAutoDetect(getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, fieldVisibility = JsonAutoDetect.Visibility.ANY)
-public class PipPerson {
-
-    private Person person;
-
-    public Person getPerson() {
-        return person;
-    }
-
-    public void setPerson(Person person) {
-        this.person = person;
-    }
+public record PipPerson(Adressebeskyttelse[] adressebeskyttelse) {
```

Comment:
Hadde kanskje gitt meining å kalle den PipPersonData sidan den ikkje inneholder noko opplysninger som tilseier kven det er snakk om

Reply:
Godt poent! Eg tenker vi likevel lar den fortsatt heite PipPerson, sidan det er det PDL kaller klassen for.